@types/commercetools-uikit__design-system/index.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module '@commercetools-uikit/design-system' {
+  export const customProperties: {
+    [key: string]: string;
+  };
+}
```

Comment:
doing a `yarn typecheck` yielded errors, so I added this.
My intuition is that it doesn't make sense to do this, when the package itself was written in TS.
but I had a brainfart and couldn't figure out where to start addressing this properly.

Reply:
I can take a look, but this file should be removed.